release-notes/latest.md

```diff
@@ -0,0 +1,6 @@
+#### <sub><sup><a name="v551-note-1" href="#v551-note-1">:link:</a></sup></sub> fix
+
+* If you are using LetsEnrypt, you will want to upgrade to this version because it includes a fix for a bug where updating existing autocert/acme certificates fails
```

Comment:
let's mention the bug and this PR number for documentation best practices. Also typo.
```suggestion
* If you are using LetsEncrypt, you will want to upgrade to this version because it includes a fix for a [bug](https://github.com/concourse/concourse/issues/4326) where updating existing autocert/acme certificates fails #4395.
```